charts/pomerium/values.yaml

```diff
@@ -208,6 +208,8 @@ service:
     nameOverride: ""
 
 ingress:
+  authenticate:
+    subdomain: "authenticate"
```

Comment:
See previous comment.  `name` rather than `subdomain` makes sense.

Reply:
Also, if there are defaults in the templates, I think this can be empty.

Reply:
fixed